migrator/src/main/java/com/google/phonenumbers/migrator/CommandLineMain.java

```diff
@@ -56,6 +59,24 @@
       description = "The BCP-47 country code the given phone number(s) belong to (e.g. 44)")
   String countryCode;
 
+  @ArgGroup()
+  OptionalParameterType optionalParameter;
+
+  static class OptionalParameterType {
+    @Option(names = {"-l", "--lenientExport"},
+        description = "boolean flag specifying that text files created after the migration process"
+            + " for standard recipe --file migrations should contain the migrated version of a given"
+            + " phone number when a migration has taken place on it, regardless of whether the"
+            + " migration resulted in an invalid phone number. By default, a strict approach is used"
+            + " and when a migration is seen as invalid, the original phone number is written to file.")
```

Comment:
Maybe add that any invalid numbers are printed out afterwards so the user is aware which numbers were invalid